pkg/game/board.go

```diff
@@ -627,9 +633,13 @@ func (board *board) scoreRoadCompletion(tile elements.PlacedTile, road feature.F
 }
 
 /*
-Calculates summary score report from all roads on a tile
+Calculates summary score report from all roads on a tile.
+
+scoreOnlyRoadWithMeepleOnTile parameter if for edge case, wwhen mid game score is calculated. The same meeple can be couted twice during mid game scoring.
```

Comment:
I do not exactly follow what is the intent for this change from this comment. Also typos

Reply:
removd comment and last argument, because i found easier solution. See comment below